dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/java/de/digitalcollections/cudami/server/backend/impl/jdbi/identifiable/IdentifierRepositoryImpl.java

```diff
@@ -149,7 +171,13 @@ public void deleteByIdentifiable(UUID identifiableUuid) {
   @Override
   public Identifier findOne(String namespace, String id) {
     final String sql =
-        "SELECT * FROM " + tableName + " WHERE namespace = :namespace, identifier = :identifier";
+        "SELECT "
+            + SQL_FULL_FIELDS_ID
+            + " FROM "
+            + tableName
+            + " AS "
+            + tableAlias
+            + " WHERE namespace = :namespace, identifier = :identifier";
```

Comment:
Man könnte vor `namespace` und `identifier` noch den Alias ergänzen. (So auch bei den anderen Selects in diesem PR) Ist aber letztlich geschmackssache.